app/controllers/application_controller.rb

```diff
@@ -36,7 +38,8 @@ def routing_error
   private
 
   rescue_from 'Exception' do |exception|
-    log_error(exception)
+    # report the original 'cause' of the exception when present
+    log_exception_to_sentry(exception) unless SKIP_SENTRY_EXCEPTION_TYPES.include?(exception.class)
```

Comment:
I'm not sure if this matters, but note that there is a semantic difference here. Before, it would log all of the errors but only send the qualifying ones to Sentry. Now it will only log the qualifying ones as well. I think this is fine but worth noting.